Assets/Scripts/GameManager/Data Classes/DungeonData.cs

```diff
@@ -103,5 +103,18 @@ public NPCData getNPCData(int index)
             return null;
         }
     }
+
+    public bool isActive()
+    {
+        return active;
+    }
+
+    public void npcCompleted(int number)
+    {
+        if (number < npcs.Length)
+        {
+            npcs[number].setHasBeenTalkedTo(true);
+        }
+    }
```

Comment:
```suggestion
    public void npcCompleted(int index)
    {
        if (number < npcs.Length)
        {
            npcs[index].setHasBeenTalkedTo(true);
        }
    }
```
In the overworld backend we talk about indexes